src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/ConfirmOrganizationUserCommand.cs

```diff
@@ -208,12 +208,15 @@ private async Task ValidateTwoFactorAuthenticationPolicyAsync(User user, Guid or
         }
     }
 
-    private async Task DeleteAndPushUserRegistrationAsync(Guid organizationId, Guid userId)
+    private async Task DeleteAndPushUserRegistrationAsync(Guid organizationId, IEnumerable<Guid> userIds)
     {
-        var devices = await GetUserDeviceIdsAsync(userId);
-        await _pushRegistrationService.DeleteUserRegistrationOrganizationAsync(devices,
-            organizationId.ToString());
-        await _pushNotificationService.PushSyncOrgKeysAsync(userId);
+        foreach (var userId in userIds)
+        {
+            var devices = await GetUserDeviceIdsAsync(userId);
+            await _pushRegistrationService.DeleteUserRegistrationOrganizationAsync(devices,
+                organizationId.ToString());
+            await _pushNotificationService.PushSyncOrgKeysAsync(userId);
```

Comment:
At a later point, we might want to come through and allow a bulk retrieval of DeviceIds by a list of user Ids within a single org.

We could then thread out the `PushSyncOrgKeysAsync(...)` call and let the backgrounds threads run.

The current implementation is fine for now and not part of this scope.